src/services/auth/auth.ts

```diff
@@ -15,15 +15,15 @@ export const login = async (
     password,
   });
 
-  const accessToken = data.data.access_token;
+  const accessToken = data.access_token;
```

Comment:
The destructured `data` property from the axios response does not directly hold `access_token`. It holds an object { status: string, data: { access_token: ..., }}. `data.data` is necessary to access that property (see current development version).